shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
`ls = require("./local_storage.js").ls;` assigns to the global variable `ls`. Assuming the `ls` variable has some property assigned to it already, like `ls.hello = "world";`, then the `require` statement doesn't clobber that property?